### PR TITLE
Tic tac toe game code in basic python

### PR DESCRIPTION
**Pull request by snehagupta-4**
This code involves basic python syntax without the use of any external libarary.
Please review it.

### CODE REVIEW

**Comment by Neilblaze** [comment]
@snehagupta-4 Good job! But your files are facing merge conflict issues. Could you please upstream & rebase your forked repo & then create the PR?

@ankitdobhal can you authorize the writing right to me so that I can resolve the conflicts that are being generated? Else, you can take this on your own :)

**Review by Neilblaze — APPROVED** [review]
Code looks good just remove the workspace.xml file.
Great job :tada: 
LGTM :+1:

File: Basic-Scripts/.idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Neilblaze) [inline Basic-Scripts/.idea/workspace.xml]: Sneha can you please remove this workspace.xml file from the PR? This infact is an autogenerated file which is creating the merge conflict issue!

**Review by Neilblaze — CHANGES_REQUESTED** [review]

**Review by Neilblaze — APPROVED** [review]

File: Basic-Scripts/.idea/workspace.xml (hunk 1 above)
Comment (snehagupta-4) [reply Basic-Scripts/.idea/workspace.xml]: Yes, I'm doing it right now

**Comment by powerexploit** [comment]
> @snehagupta-4 Good job! But your files are facing merge conflict issues. Could you please upstream & rebase your forked repo & then create the PR?
> 
> @ankitdobhal can you authorize the writing right to me so that I can resolve the conflicts that are being generated? Else, you can take this on your own :)

No issue I will resolve the conflict @Neilblaze .

**Review by snehagupta-4 — COMMENTED** [review]
Could you please help me how to do this? I'm new to Github so don't know properly about it

File: Basic-Scripts/.idea/workspace.xml (hunk 1 above)
Comment (powerexploit) [reply Basic-Scripts/.idea/workspace.xml]: > This code involves basic python syntax without the use of any external libarary.
> Please review it.

@snehagupta-4 nice and clean code.

**Comment by powerexploit** [comment]
> Could you please help me how to do this? I'm new to Github so don't know properly about it

@snehagupta-4 These are some resources will help you out to resolve this issue.
But you  find difficulties then me @ankitdobhal will resolve that.
Try do it by yourself.

- https://opensource.com/article/20/4/git-merge-conflict
- https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

Thanks

File: Basic-Scripts/.idea/workspace.xml (hunk 1 above)
Comment (snehagupta-4) [reply Basic-Scripts/.idea/workspace.xml]: > > This code involves basic python syntax without the use of any external libarary.
> > Please review it.
> 
> @snehagupta-4 nice and clean code.

Thank you so much sir

**Comment by snehagupta-4** [comment]
> > Could you please help me how to do this? I'm new to Github so don't know properly about it
> 
> @snehagupta-4 These are some resources will help you out to resolve this issue.
> But you find difficulties then me @ankitdobhal will resolve that.
> Try do it by yourself.
> 
> * https://opensource.com/article/20/4/git-merge-conflict
> * https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts
> 
> Thanks

I'm so sorry sir, but I'm not getting hoe to solve the issue. Please help.

**Comment by Neilblaze** [comment]
@snehagupta-4 I think you haven't deleted the workspace.xml file from the PR! Do it & rest of the task will be maintained by @ankitdobhal

**Comment by powerexploit** [comment]
@snehagupta-4 closing this PR.

**Pull request closed without merging**